ui/panels-plugin/src/plugins/time-series-chart/TimeSeriesChartPanel.tsx

```diff
@@ -55,7 +55,6 @@ export function TimeSeriesChartPanel(props: TimeSeriesChartProps) {
 
   // convert Perses dashboard format to be ECharts compatible
   const yAxis = {
-    name: y_axis?.label ?? '',
```

Comment:
Decided to change the axis label to a regular component instead of using the out of the box ECharts one, it gives us more control and doesn't have the limitations of being rendered to canvas (aka will be easier to test)